client/src/components/content/ModelerSidebar/ModelerSidebar.jsx

```diff
@@ -10,7 +10,7 @@ import {
   applicationChangedHandler,
   taskChangedHandler,
   inputParameterChangeHandler,
-  outputVarNameChangeHandler,
+  outputValNameChangeHandler,
```

Comment:
I see that the abbreviation has been used before, but I would still argue for using the long form of value :) 
```suggestion
  outputValueNameChangeHandler,
```